mongoengine/queryset/base.py

```diff
@@ -1198,6 +1215,20 @@ def read_preference(self, read_preference):
         queryset._cursor_obj = None  # we need to re-create the cursor object whenever we apply read_preference
         return queryset
 
+    def read_concern(self, read_concern):
```

Comment:
sorry I forgot to flag this earlier but use a dict here as well for read_concern. Alternative could be to have a utility function and accept both dict and ReadConcern (and do the same for WriteConcern)

Reply:
I think the way it is now is fine as the only way to set the write concern is on update operations. I've change the interface to accept a dictionary which is used to create a ReadConcern instance.